recipe/conda_build_config.yaml

```diff
@@ -29,10 +19,46 @@ cxx_compiler:                  # [win]
   - vs2015                     # [win]
 CONDA_BUILD_SYSROOT:           # [osx]
   - /opt/MacOSX10.9.sdk        # [osx]
+macos_min_version:
+  - 10.9
+macos_machine:
+  - x86_64-apple-darwin13.4.0
```

Comment:
@msarahan, are the 3 options above used? They are removed by conda-smithy

Reply:
they will be used when you start using the Anaconda compilers.  Currently, they are not.

Reply:
Then I guess we have to mark them as used in conda-smithy right? Or will they be marked as used automatically if the recipe used new compilers?

Reply:
they'll probably need to be added to some kind of "always used" list either in conda-smithy or in conda-build.  I don't know if they actually get detected, or if they're more indirect.

Reply:
@msarahan, what should `CONDA_BUILD_SYSROOT` be set to?

Reply:
You have to install the appropriate OSX SDK.  The value of CONDA_BUILD_SYSROOT is where you have that extracted.

Reply:
@jakirkham, do you know where it is installed in travis-ci?

Reply:
Here's one place you can get them: https://github.com/phracker/MacOSX-SDKs

Reply:
We will be updating to 10.10 sometime soon, I think - but we'll give you a heads-up when we're going to do it.

Reply:
I guess we can install it into `/opt/MacOSX10.9.sdk` which is the value set here currently.